### PR TITLE
update design,make search sticky in medium devices,add default collapse.

### PR DESCRIPTION
**Pull request by hanzalakhan809**
if every thing looks ok, don't merge the brach i will merge it after merging its backend part.

### CODE REVIEW

File: components/template/Templates.vue
Comment (dhaladitya108) [inline components/template/Templates.vue:1]: one page should have only have one h1, two h1 are present here one inside the hero, other with the 'heading' class

File: components/template/Templates.vue [hunk 1]
```diff
@@ -118,15 +119,12 @@ export default {
...
-  .left-sidebar {
-    min-height: auto;
-    box-shadow: 0px 4px 8px 0px rgba(0, 0, 0, 0.04);
-    position: sticky;
-    top: 54px;
-    padding: 1rem 0;
-  }
   .heading {
-    margin-top: 12px;
+    display: none;
```
Comment (dhaladitya108) [inline components/template/Templates.vue:123]: let this heading also be here for smaller screen sizes as well. No need to make it display: none;

File: pages/templates/index.vue [hunk 2]
```diff
@@ -30,6 +30,62 @@ export default {
...
+          created_at: '2023-10-20T13:45:21.030Z',
+          updated_at: '2023-10-20T15:48:55.836Z',
+          slug: 'donation-forms',
+          kind: 'generic',
+        },
+        {
+          id: 5,
+          name: 'Organization Forms',
+          is_active: true,
+          created_at: '2023-10-20T13:45:21.071Z',
```
Comment (dhaladitya108) [inline pages/templates/index.vue:59]: I believe these all were for testing purposes. Should it remain in production as well?

Comment (hanzalakhan809) [reply pages/templates/index.vue:59]: No it is only for testing, once backend was deployed i remove this.

File: pages/templates/categories/_slug.vue [hunk 3]
```diff
@@ -19,18 +19,75 @@ export default {
...
+        {
+          id: 23,
+          name: 'Order Forms',
+          is_active: true,
+          created_at: '2023-10-20T13:45:21.191Z',
+          updated_at: '2023-10-20T15:42:32.427Z',
+          slug: 'order-forms',
+          kind: 'industry',
+        },
+      ],
```
Comment (dhaladitya108) [inline pages/templates/categories/_slug.vue:36]: same here as well should it remain here?

Comment (hanzalakhan809) [reply pages/templates/categories/_slug.vue:36]: No, it will be removed also

File: components/template/TemplateCategories.vue [hunk 4]
```diff
@@ -53,12 +79,18 @@ export default {
   data() {
     return {
       showCategories: true,
-      isCollapsed: false,
+      isCollapsed: { industry: true }, // Hardcoded industry to open industry collapse by default
+      isCollapsedSm: false,
     }
   },
+  methods: {
+    toggleCollapse(kind) {
```
Comment (dhaladitya108) [inline components/template/TemplateCategories.vue:87]: can you use some better naming convention here. Kind sounds much more generic here. You can use categoryType or categoryGroup.

**Pull request merged**